frontend_tests/node_tests/upload.js

```diff
@@ -284,6 +284,7 @@ run_test('file_input', () => {
     const event = {
         target: {
             files: files,
+            value: "C:\fakepath\portland.png",
```

Comment:
Is it worth adding a little test here for uploading the same file twice?  Might be worth doing in a follow-up commit.